tests/test_fips.py

```diff
@@ -239,12 +239,21 @@ def test_gcrypt_binary(container_per_test: ContainerData) -> None:
 
     """
 
-    container_per_test.connection.check_output(
-        "zypper --gpg-auto-import-keys -n ref && zypper -n in gcc libgcrypt-devel && zypper -n clean && "
+    c = container_per_test.connection
+
+    c.check_output(
+        "zypper --gpg-auto-import-keys -n ref && "
+        "zypper -n in gcc libgcrypt-devel dirmngr && "
+        "zypper -n clean && "
```

Comment:
The clean is not necessary, the container is discarded after the test. It just makes the test run longer
```suggestion
```